server/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/SimpleModel.java

```diff
@@ -84,7 +84,11 @@ public MovAvgModel clone() {
 
     @Override
     public double next(Collection<Double> values) {
-        return MovingFunctions.simpleMovAvg(values);
+        double avg = 0;
```

Comment:
can we call this `sum` as its not actually the `avg`? 😉

Reply:
durp.  :)